.github/workflows/push.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         run: npx serverless deploy --stage dev --param="resource-suffix=${{ secrets.DEPLOYMENT_SUFFIX_DEV }}"
 
       - name: get deployment url
-        run: echo BASE_URL=$(npx serverless info | grep -o 'https.*\/dev\/') >> "$GITHUB_ENV"
+        run: echo BASE_URL=$(npx serverless info --stage dev | grep -o 'https.*\/dev\/') >> "$GITHUB_ENV"
```

Comment:
Do you need the grep if you are specifying the stage as `dev`?

Reply:
Yep, the full output looks like this:

```console
$ npx serverless info --stage dev
service: qgis-plugin-repo
stage: dev
region: ap-southeast-2
stack: qgis-plugin-repo-dev
endpoints:
  ANY - https://j4osj0imtb.execute-api.ap-southeast-2.amazonaws.com/dev
  ANY - https://j4osj0imtb.execute-api.ap-southeast-2.amazonaws.com/dev/{proxy+}
functions:
  app: qgis-plugin-repo-dev-app
```

Reply:
Oh. Okay.